.github/workflows/noop-ci.yml

```diff
@@ -1,45 +1,43 @@
+# The jobs in this workflow are necessary because they are configured as
+# required checks on PRs. Each job actually has a corresponding "real" job
+# definition in a different workflow file:
+#
+# codegen-ci.yml: Codegen Tests
+# tssdk-ci.yml: Typescript Tests
+# python-ci.yml: Python Tests
+#
+# By hard-coding passing jobs with the same name here every PR will satisfy the
+# repo's "required checks" EXCEPT those that trigger the "real" job AND that
+# "real" job fails.
+#
+# It will look a little confusing: any PR that triggers a corresponding "real"
+# job will have duplicate check entries, one that passes (this noop job) and the
+# "real" job that either passes or fails.
 name: Noop CI
 on:
-  # This workflow is intended to be manually run on a branch for which the
-  # PR triggers no other CI workflow (and thus cannot trigger any of the
-  # required jobs). Eg. docs, examples, new packages that don't have a CI
-  # workflow setup yet.
-  workflow_dispatch:
```

Comment:
this was a failed experiment: according to the docs I should have been able to create a "check-run" on a commit that is linked to an open PR and have that check register on the PR itself. I even verified that the checks I created had associated `pull_request` objects but the GitHub UI never showed the checks :-( bogus advertising